### PR TITLE
PWX-23472-update px version to 2.1.9.4

### PR DESCRIPTION
**Pull request by dahuang-purestorage**
Signed-off-by: dahuang <dahuang@purestorage.com>

**What this PR does / why we need it**:
Update px version to 2.9.1.4
**Which issue(s) this PR fixes** (optional)
Closes # PWX-23472

**Special notes for your reviewer**:
Tested deployment locally.

### CODE REVIEW

**Comment by dahuang-purestorage** [comment]
do we need to update storkVersion as well ?

**Review by ggriffiths — APPROVED** [review]

**Review by trierra — APPROVED** [review]

File: charts/portworx/values.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 deploymentType: oci                     # accepts "oci" or "docker"
 imageType: none                         #
-imageVersion: 2.8.1.2                   # Version of the PX Image.
+imageVersion: 2.9.1.4                   # Version of the PX Image.
```
[the remark this answers is not in the document]
Comment (harsh-px) [reply charts/portworx/values.yaml:5]: 2.9.1.4 is correct since we want to update  to last stable, not latest.

**Review by harsh-px — COMMENTED** [review]
Please fix PR title

**Comment by dahuang-purestorage** [comment]
> 

looking at the px-installer for 2.9.1.4. the stork version is 2.7.0, so i don't think we need to update to 2.8.0 just yet. 
https://install.portworx.com/2.9?mc=false&kbver=1.22.0&b=true&c=px-cluster-5b11a15d-d2d0-480d-92c9-9a1d300891a8&stork=true&csi=true&mon=true&tel=false&st=k8s

**Pull request merged**